ouroboros-consensus-cardano/src/byron/Ouroboros/Consensus/Byron/Node.hs

```diff
@@ -8,18 +8,19 @@
 
 module Ouroboros.Consensus.Byron.Node (
     PBftSignatureThreshold (..)
-  , ProtocolParamsByron (..)
   , blockForgingByron
   , byronBlockForging
-  , defaultPBftSignatureThreshold
-  , mkByronConfig
-  , protocolClientInfoByron
-  , protocolInfoByron
     -- * Secrets
   , ByronLeaderCredentials (..)
   , ByronLeaderCredentialsError
   , mkByronLeaderCredentials
   , mkPBftCanBeLeader
+    -- * ProtocolInfo
+  , ProtocolParams (..)
+  , defaultPBftSignatureThreshold
+  , mkByronConfig
+  , protocolClientInfoByron
+  , protocolInfoByron
```

Comment:
INFO: introduced some sectioning here to group definitions related to `ProtocolInfo`